### PR TITLE
runs rubocop autofix

### PR DESCRIPTION
**Pull request by warp**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -41,13 +40,15 @@ def auto_complete
     session[:auto_complete] = controller_name.to_sym
     respond_to do |format|
       format.any(:js, :html)   { render partial: 'auto_complete' }
-      format.json { render json: @auto_complete.inject({}){|h,a|
-        h[a.id] = a.respond_to?(:full_name) ? h(a.full_name) : h(a.name); h
-      }}
+      format.json do
+        render json: @auto_complete.inject({}){|h, a|
+                       h[a.id] = a.respond_to?(:full_name) ? h(a.full_name) : h(a.name); h
```
Comment (UncommonDevelopment) [inline app/controllers/application_controller.rb]: Stop

Sent from my iPhone

On Jan 19, 2015, at 3:17 PM, "Hound" <notifications@github.com<mailto:notifications@github.com>> wrote:


In app/controllers/application_controller.rb<https://github.com/fatfreecrm/fat_free_crm/pull/408#discussion-diff-23187392>:

> @@ -41,13 +40,15 @@ def auto_complete
>      session[:auto_complete] = controller_name.to_sym
>      respond_to do |format|
>        format.any(:js, :html)   { render partial: 'auto_complete' }
> -      format.json { render json: @auto_complete.inject({}){|h,a|
> -        h[a.id] = a.respond_to?(:full_name) ? h(a.full_name) : h(a.name); h
> -      }}
> +      format.json do
> +        render json: @auto_complete.inject({}){|h, a|
> +                       h[a.id] = a.respond_to?(:full_name) ? h(a.full_name) : h(a.name); h


Line is too long. [90/80]
Do not use semicolons to terminate expressions.

**Comment by steveyken** [comment]
This is fine to merge, but I might turn off hound as they seem to disagree over what standards should be applied :)

**Comment by warp** [comment]
I think we can turn off string literal and line length complaints some of
the others are usefull

File: app/controllers/admin/field_groups_controller.rb [hunk 2]
```diff
@@ -64,7 +63,7 @@ def sort
     field_group_ids = params["#{asset}_field_groups"]
 
     field_group_ids.each_with_index do |id, index|
-      FieldGroup.update_all({position: index+1}, {id: id})
+      FieldGroup.update_all({ position: index + 1 }, id: id)
```
Comment (warp) [inline app/controllers/admin/field_groups_controller.rb]: oops just spotted a bug here should be FieldGroup.where(id: id).update_all(position: index + 1)

File: app/controllers/admin/fields_controller.rb [hunk 3]
```diff
@@ -86,7 +85,7 @@ def sort
     field_ids = params["fields_field_group_#{field_group_id}"] || []
 
     field_ids.each_with_index do |id, index|
-      Field.where(id: id).update_all(position: index+1, field_group_id: field_group_id)
+      Field.where(id: id).update_all(position: index + 1, field_group_id: field_group_id)
```
Comment (warp) [inline app/controllers/admin/fields_controller.rb]: also a bug here

**Pull request merged**